engine/src/main/scala/cromwell/engine/language/CromwellLanguages.scala

```diff
@@ -0,0 +1,44 @@
+package cromwell.engine.language
+
+import cromwell.engine.language.CromwellLanguages._
+import cromwell.engine.workflow.lifecycle.materialization.LanguageFactory
+
+/**
+  * Provides a global singleton access to the instantiated language factories.
+  * @param languageConfig A set of language configuration elements.
+  */
+case class CromwellLanguages(languageConfig: List[LanguageConfigurationEntry]) {
+
+  val Languages: Map[CromwellLanguageName, LanguageVersions] = (languageConfig map { lc =>
```

Comment:
Since this is not an `object` it should be lower `l`

Reply:
Is that the rule? I never actually worked out when to use Upper vs lower. `val x: <some object>` works for me!